packages/fhir-converter/Dockerfile

```diff
@@ -10,9 +10,9 @@ RUN npm install --only=production --no-fund --no-optional --no-audit
 ### Prod Image                         ##
 #########################################
 FROM node:18-slim
-RUN  apt-get update && apt install libcurl4-gnutls-dev -y && apt autoremove -y
+# RUN  apt-get update && apt install libcurl4-gnutls-dev -y && apt autoremove -y
```

Comment:
this is no longer needed since we cleaned up the converter - leaving in just in case for later